presto-main/pom.xml

```diff
@@ -17,6 +17,14 @@
     </properties>
 
     <dependencies>
+        <!-- this is some weird stuff -->
+        <dependency>
```

Comment:
You accidentally imported a Guava class from jersey.repackaged. It's easy to do this by accident when the IDE prompts for an import. When this happens, and there obvious things that you never want to import, and an exclusion for that package. (My exclusion list has grown to hundreds of entries)=